src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('tbody');
+
+const listHeader = [ ...document.querySelectorAll('th') ];
+// console.log(listHeader)
+listHeader.map((item, index) => {
+  // debugger
+  item.addEventListener('click', () => {
```

Comment:
let's put an event listener to the whole table header (use event delegation for this)

Reply:
if you still want to know the index of each heading, use `td.cellIndex` prop